Detector/DetFCChhHCalTile/compact/FCChh_HCalBarrel_TileCal.xml

```diff
@@ -2,30 +2,34 @@
 <lccdd xmlns:compact="http://www.lcsim.org/schemas/compact/1.0"
   xmlns:xs="http://www.w3.org/2001/XMLSchema"
   xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/compact/1.0/compact.xsd">
-
   <info name="FCChh_HCalBarrel_TileCal"
-    title="Barrel Calorimeters"
-    author="C. Solans, J. Lingemann"
+    title="Barrel Calorimeter"
+    author="C. Neubueser"
     url="no"
     status="development"
     version="1.0">
-    <comment>HCal layout based on ATLAS HCal</comment>
+    <comment>HCal layout based on ATLAS HCal, with realistic longitudinal segmentation and steel support</comment>
   </info>
-  
   <define>
-    <!-- 20cm for support is reduced to 18cm, 1cm is needed to place max number of trapazoids and 1cm needed to compensate for faceplate-->
-    <constant name="BarHCal_rmaxWithBuffer" value="BarHCal_rmax-180*mm"/>
+    <!-- 21.2cm for support, 1cm is needed to place max number of trapazoids and 1cm needed to compensate for faceplate-->
+    <constant name="BarHCal_rmaxWithBuffer" value="BarHCal_rmax-212*mm"/>
```

Comment:
Ok, let me try to clarify. First there was some buffer space between the HCAL and the solenoid for the steel girder (the mechanical support structure to connect the HCAL modules). These module support structure is now implemented in the HCAL construction. The buffer is not needed any longer. Therefore the BarHCal_rmaxWithBuffer was not used when I pushed this PR, I simply forgot to delete it.